### PR TITLE
[RFR] GitHub login button

### PR DESCRIPTION
**Pull request by alexisjanvier**

### CODE REVIEW

**Comment by alexisjanvier** [comment]
@jpetitcolas @floo51 Missing some tests, but RFR

File: docs/config/development-dist.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default {
+    githubAppId: 'xxx',
+    githubScopes: [
```
Comment (jpetitcolas) [inline docs/config/development-dist.js]: Why do you need to repeat it?

File: docs/config/production-dist.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export default {
+    appBaseUrl: 'http://localhost:8080',
+    githubAppId: 'xxx',
+    githubRedirection: '/setup.html',
+    githubScopes: [
```
Comment (jpetitcolas) [inline docs/config/production-dist.js]: Same

**Comment by jpetitcolas** [comment]
Merged, but please fix my comments in your next PR.:)

**Pull request merged**